### PR TITLE
document new registration procedure

### PR DESCRIPTION
**Pull request by jdries**
Documentation on registration still mentions early adopters, and update to reflect the more simple eoplaza procedure.

### CODE REVIEW

**Comment by jdries** [comment]
@bschumac  @dthiex  this will be new registration procedure, feel free to review already, but it would also be convenient if we just merge the page without adding it to navigation, so we can properly see the rendered version?

**Comment by m-mohr** [comment]
You can see the rendered version as follows:
https://github.com/openEOPlatform/documentation/blob/main/DEVELOPMENT.md

File: join/free_trial.md
Comment (bschumac) [inline join/free_trial.md:1]: Shouldn't this be:
https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://openeo.cloud/&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid

Comment (jdries) [reply join/free_trial.md:1]: that was indeed the original, but it's better when users directly see the dashboard where they can modify their account. Hence I now adjusted to:
https://portal.terrascope.be/dashboard

**Pull request merged**